examples/Language_Translation/English_to_French_Translation/requirements.txt

```diff
@@ -0,0 +1,6 @@
+fastBPE
+regex 
+requests 
+sacremoses 
+subword_nmt
+fairseq-0.9.0.tar.gz
```

Comment:
@AshwinChafale Is there any reason for not using the public pypi version of fairseq?

Reply:
PyPI version of the [fairseq](https://pypi.org/project/fairseq/) hasn't been updated since Dec 4, 2019. And our model is using the updated fairseq classes and modules that are not present in the PyPi version of fairseq.